lib/fizzy/execute.cpp

```diff
@@ -378,6 +378,55 @@ execution_result execute(Instance& instance, FuncIdx func_idx, std::vector<uint6
             labels.push_back(label);
             break;
         }
+        case Instr::if_:
+        {
+            if (static_cast<uint32_t>(stack.pop() != 0))
+            {
+                const auto arity = read<uint8_t>(immediates);
+                const auto target_pc = read<uint32_t>(immediates);
+                const auto target_imm = read<uint32_t>(immediates);
+                read<uint32_t>(immediates);  // Omit the immediates for else instruction.
```

Comment:
I would instead just do `immediates += 2 * sizeof(uint32_t)`, there's no need to call `memcpy` here.

(also better to call it "skip" instead of "omit")